### ISSUE TITLE
Incorrect auto-custos on clef change at the end of a line

### ISSUE THREAD

**Issue by henryso** [gregorio-project/gregorio#569]
The automatic custos between the two pictured lines was generated with `(z0::c3)`.  Because of the clef change, the custos should be showing on the top line, but it is not.

![image](https://cloud.githubusercontent.com/assets/421318/9371267/77101f9a-46a5-11e5-8c87-414b18f7ed98.png)

The generated GregorioTeX code is correct:
```
\GreDiscretionary{%
\GreSyllable{}{}{}{1}{\GreSetNextSyllable{}{}{}}{}{0}{}{%
\GreInDivisioFinalis{}%
}%
%
}{%
\GreSyllable{}{}{}{1}{\GreSetNextSyllable{}{}{}}{}{0}{}{%
\GreEndOfElement{1}{1}%
\GreCusto{12}%
\GreInDivisioFinalis{}%
\GreChangeClef{c}{3}{0}{3}%
}%
%
}%
```

Notably, the `\GreCusto{12}` does indicate that the custos should be on the top line.  It would appear that the "localrightcontent" supplants this setting with one based on the physical position of the next note.

**Comment by henryso** [comment]
What's happening is that the computation for the "next note pitch" in Gregorio is not considering the automatic custos created by `z0`.  However, after accounting for that pitch, it needs to be undone if the discretionary does print before the end of a line (i.e, with 'Z', which was luckily covered by the alternate clef test).

**Comment by eroux** [comment]
fix by #571
